From d35e773d64a0b00b0f1bbf97cd9bf7eed75a9d30 Mon Sep 17 00:00:00 2001
From: Jakub Kadlcik <frostyx@email.cz>
Date: Wed, 1 Nov 2023 10:11:01 +0100
Subject: [PATCH] createrepo_mod: fix failing tests on F39+ because of new
 compresssion

Fix RHBZ 2247292

On F39+ we use `zst` instead of `gz`
https://fedoraproject.org/wiki/Changes/createrepo_c_1.0.0
---
 tests/test_createrepo_mod/test_createrepo_mod.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_createrepo_mod/test_createrepo_mod.py b/tests/test_createrepo_mod/test_createrepo_mod.py
index a38eaea..0d93f49 100644
--- a/tests/test_createrepo_mod/test_createrepo_mod.py
+++ b/tests/test_createrepo_mod/test_createrepo_mod.py
@@ -48,6 +48,6 @@ def test_run_modifyrepo(test_output_dir):
                     os.path.join(test_output_dir, "modules.yaml"))
 
     assert os.path.isfile(os.path.join(test_output_dir, "modules.yaml"))
-    retval = run_modifyrepo(test_output_dir)
+    retval = run_modifyrepo(test_output_dir, compress_type="gz")
     assert glob.glob(os.path.join(test_output_dir, "repodata", "*-modules.yaml.gz"))
     assert retval == 0
-- 
2.41.0

